Ch setup database 163623420

Bobjayafam opened the pull request:
#### What does this PR do?
Setup database to ensure persistence of data
#### Description of Task to be completed?
- install npm package npm
- setup database connection
- configure script for running database migrations
- configure script for seeding data into the database
#### How should this be manually tested?
- clone the GitHub repo
- checkout branch ``ch-setup-database-163623420``
- run ``npm install`` to install dependencies
- installl postgresql and create a database and add the connection string to .env as DATABASE_URL
- run ``npm run migration`` to create tables
- run ``npm run seed`` to seed sample data into the database
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/163623420

Bobjayafam committed:
chore(server): setup database

- install pg
- create a db folder

[Finishes #163623420]

Bobjayafam committed:
chore(server): setup database

- install pg
- create a db folder

[Finishes #163623420]

Bobjayafam committed:
chore(server): set up postgres database

- create database
- configure pg connection
- configure a script for running migrations
- configure a script for running seed

[Finishes #163623420]

Bobjayafam committed:
Merge branch 'ch-setup-database-163623420' of https://github.com/Bobjayafam/politico into ch-setup-database-163623420

akinwale-habibullah commented on server/db/connection.js:
```diff
@@ -0,0 +1,14 @@
+dotenv.config();
+
+
+const connectionString = process.env.DATABASE_URL;
```
Please add this environment variable to your .env.sample file. and also this detail in your README file under the installing section @Bobjayafam

akinwale-habibullah reviewed (commented):
In your .env.example file, add variables for the admin user, which you then use to seed your database for the initial admin account. Please make these changes and revert @Bobjayafam

Bobjayafam committed:
chore(server): setup database

- seed admin data
- update README
- add credentials to .env file

[Finishes #ch-setup-database-163623420]

Bobjayafam commented:
@akinwale-habibullah, the changes have been effected
